docs/source/usage/plugins/checkpoint.rst

```diff
@@ -21,18 +21,17 @@ What is the format of the created files?
 
 We write our fields and particles in an open markup called :ref:`openPMD <pp-openPMD>`.
 
-For further details, see the according sections in :ref:`the openPMD API <usage-plugins-openPMD>` and :ref:`ADIOS <usage-plugins-ADIOS>`.
+For further details, see the according section in :ref:`the openPMD API <usage-plugins-openPMD>`.
 
 External Dependencies
 ^^^^^^^^^^^^^^^^^^^^^
 
-The plugin is available as soon as the :ref:`openPMD API or ADIOS libraries <install-dependencies>` are compiled in.
+The plugin is available as soon as the :ref:`openPMD API library <install-dependencies>` are compiled in.
 
 .cfg file
 ^^^^^^^^^
 
 You can use ``--checkpoint.period`` to specify the output period of the created checkpoints.
-Note that this plugin will only be available if the openPMD API, libSplash (HDF5) or ADIOS is found during compile configuration.
```

Comment:
Why is this note totally gone? Should it not state:

> Note that this plugin will only be available if the openPMD API is found during compile configuration.

Reply:
we have a similar comment two lines above.